lib/templates/_has_more_docs.html

```diff
@@ -0,0 +1,3 @@
+{{#hasMoreThanOneLineDocs}}
+<a href="{{href}}">&hellip;</a>
```

Comment:
Interesting - did the … char not make it through the various file encodings?

You could also remove the `a href`. If just the line is long and gets truncated by the browser, it'll show an ellipses from the text-overflow css style. So, a plain ellipses char from a 2nd paragraph would match that.